pkg/client/k8s.go

```diff
@@ -553,9 +553,11 @@ func (k K8sClient) GetCSPIs(cspiNames []string, labelselector string) (*cstorv1.
 	for _, name := range cspiNames {
 		if pool, ok := poolMap[name]; ok {
 			list = append(list, pool)
-		} else {
-			fmt.Printf("Error from server (NotFound): pool %s not found\n", name)
 		}
+		// else {
```

Comment:
This writes a log to stdout/err when the storage is _guessed_ by brute force, which isn't very optimal for good CLI UX.
E.g. If a user is trying to describe an LVM volume, it's not optimal to see `Error from server, CStor pool lvm-vol1 not found`